.github/workflows/stage_release.yaml

```diff
@@ -47,9 +47,9 @@ on:
         required: false
         default: 'main'
       website-repository:
-        description: 'The URL of the website repository, e.g. git@github.com:kroxylicious/kroxylicious.github.io.git'
+        description: 'The URL of the website repository, starting with everything after the @, e.g. github.com/kroxylicious/kroxylicious.github.io.git'
```

Comment:
nit - its not a URL.  It is a host and path.

Reply:
Agreed, changed from URL to URI as it identifies a resource